File: backend/src/main/java/space/davids_digital/vk_gpt_bot/WebApp.java
package space.davids_digital.vk_gpt_bot;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.security.servlet.UserDetailsServiceAutoConfiguration;

@SpringBootApplication(exclude = {UserDetailsServiceAutoConfiguration.class})
public class WebApp {
    public static void main(String[] args) {
        SpringApplication.run(WebConfig.class, args);
    }
}

File: backend/src/main/java/space/davids_digital/sweetie/WebApp.kt
package space.davids_digital.sweetie

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.security.servlet.UserDetailsServiceAutoConfiguration

fun main(args: Array<String>) {
    SpringApplication.run(WebConfig::class.java, *args)
}

@SpringBootApplication(exclude = [UserDetailsServiceAutoConfiguration::class])
class WebApp
